components/ActionSheet/examples/ActionSheetTypicalUseExampleViewController.m

```diff
@@ -25,51 +29,37 @@ @interface ActionSheetTypicalUseExampleViewController ()
 
 @end
 
-@implementation ActionSheetTypicalUseExampleViewController {
-  MDCButtonScheme *_buttonScheme;
-}
+@implementation ActionSheetTypicalUseExampleViewController
 
 - (instancetype)init {
   self = [super init];
   if (self) {
     self.title = @"Action Sheet";
-    _colorScheme =
-        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-    _typographyScheme = [[MDCTypographyScheme alloc] init];
+    _containerScheme = [[MDCContainerScheme alloc] init];
+    _containerScheme.colorScheme =
+        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];;
     _showButton = [[MDCButton alloc] init];
-    _buttonScheme = [[MDCButtonScheme alloc] init];
   }
   return self;
 }
 
-- (MDCContainerScheme *)containerScheme {
-  if (!_containerScheme) {
-    _containerScheme = [[MDCContainerScheme alloc] init];
-  }
-  _containerScheme.colorScheme = self.colorScheme;
-  _containerScheme.typographyScheme = self.typographyScheme;
-  return _containerScheme;
-}
-
 - (void)viewDidLoad {
   [super viewDidLoad];
 
-  self.view.backgroundColor = _colorScheme.backgroundColor;
-  [_showButton setTitle:@"Show action sheet" forState:UIControlStateNormal];
-  [_showButton sizeToFit];
-  _buttonScheme.colorScheme = _colorScheme;
-  _buttonScheme.typographyScheme = _typographyScheme;
-  [MDCContainedButtonThemer applyScheme:_buttonScheme toButton:_showButton];
-  [_showButton addTarget:self
+  self.view.backgroundColor = self.containerScheme.colorScheme.backgroundColor;
```

Comment:
Please add a conditional check and default value for `colorScheme`.

Reply:
Fixed